### ISSUE TITLE
Users are unable to reset passwords

### ISSUE THREAD

**Issue by rcheetham** [azavea/opendataphilly-ckan#100]
Two users have reported that they are unable to reset their passwords and they receive the following error:

Could not send reset link: SMTPAuthenticationError(535, '5.7.8 Username and Password not accepted. Learn more at\n5.7.8 https://support.google.com/mail/?p=BadCredentials w34sm2306200qth.81 - gsmtp')

I attempted to reset my own password using "rcheetham" and received the same error.

**Comment by rbreslow** [comment]
It looks like someone changed the password to the info@opendataphilly.org account:

<img width="1743" alt="Gmail 2019-06-14 13-51-43" src="https://user-images.githubusercontent.com/1774125/59528756-1d165500-8ead-11e9-98c0-7460717e1fe8.png">

The entry in LastPass was updated, so I think there was a breakdown in communication where this should have been surfaced so we could have updated the application's configuration.

I replaced the value of `smtp.password` in `/etc/ckan/default/production.ini`, and restarted the `apache2` service.

After that, I was able to request a password reset:

<img width="875" alt="image" src="https://user-images.githubusercontent.com/1774125/59528867-69619500-8ead-11e9-8bb0-a94945210ecf.png">

I made sure to update the `group_vars/staging` file on the file share as well.
